fe/src/main/java/org/apache/doris/catalog/EsTable.java

```diff
@@ -294,6 +331,13 @@ public void readFields(DataInput in) throws IOException {
             } else {
                 enableKeywordSniff = true;
             }
+            if (tableContext.containsKey("maxDocValueFields")) {
+                try {
+                    maxDocValueFields = Integer.parseInt(tableContext.get("maxDocValueFields"));
+                } catch (Exception e) {
+                    maxDocValueFields = 30;
```

Comment:
why set 30, not sed DEFAULT_MAX_DOCVALUE_FIELDS.

Reply:
Oh，I forget this...